### DISCUSSION TITLE
posts/2022-04-22-finality-in-consensus-protocols/

### DISCUSSION THREAD

**Discussion by giscus** [phnaharris/phnaharris.github.io#3] [Announcements]
# posts/2022-04-22-finality-in-consensus-protocols/

Tính kết thúc của một giao dịch trong một hệ thống thanh toán là một trong những yếu tố tối quan trọng để xem xét đến việc sử dụng hệ thống này trong thanh toán thực tế.

https://phnaharris.github.io/posts/2022-04-22-finality-in-consensus-protocols/

**Comment by harrispham98** [comment]
nice post
